frontend/src/css/modules/submission.scss

```diff
@@ -0,0 +1,9 @@
+/* Styles for Submission page */
+
+.remaining-submission-clock {
+	padding-top: 55px;
+}
+
+#submission {
```

Comment:
@aka-jain Actually the `col s6` sets the width to `50%`, which doesn't look good with grid of `m7` .Please refer the screenshot for the same.  That's why I changed the width to `70%`.
![shot5](https://user-images.githubusercontent.com/12206047/27983740-3af64d9e-63e1-11e7-8d57-2a63c7e0d5d4.png)